ui/src/pages/projects/projects.tsx

```diff
@@ -2,17 +2,32 @@ import { useProjects } from 'data-services/hooks/projects/useProjects'
 import { PageFooter } from 'design-system/components/page-footer/page-footer'
 import { PageHeader } from 'design-system/components/page-header/page-header'
 import { PaginationBar } from 'design-system/components/pagination-bar/pagination-bar'
+import * as Tabs from 'design-system/components/tabs/tabs'
 import { NewProjectDialog } from 'pages/project-details/new-project-dialog'
+import { useState } from 'react'
 import { STRING, translate } from 'utils/language'
 import { usePagination } from 'utils/usePagination'
 import { UserPermission } from 'utils/user/types'
+import { useUser } from 'utils/user/userContext'
 import { ProjectGallery } from './project-gallery'
-import styles from './projects.module.scss'
+
+export const TABS = {
+  USER_PROJECTS: 'user-projects',
+  ALL_PROJECTS: 'all-projects',
+}
 
 export const Projects = () => {
+  const { user } = useUser()
+  const [selectedTab, setSelectedTab] = useState(
+    user.loggedIn ? TABS.USER_PROJECTS : TABS.ALL_PROJECTS
+  )
   const { pagination, setPage } = usePagination()
+  const filters =
+    user.loggedIn && selectedTab === TABS.USER_PROJECTS
+      ? [{ field: 'public', value: 'false' }]
```

Comment:
@mohamedelabbas1996 here is where the params we pass to the backend can be tweaked